docs/site_specific_config/rocm.md

```diff
@@ -227,68 +229,45 @@ graph LR;
 
 ### C/C++ Compilers
 
-* [ROCm-LLVM](#ROCm-LLVM) (AMDGPU LLVM):
+* [ROCm-LLVM](#ROCm-LLVM) (AMDGPU LLVM / amdclang++) ([Github](https://github.com/ROCm/llvm-project)):
     * The foundation of [ROCm](#ROCm)'s compiler toolchain
     * Based on LLVM/Clang infrastructure with AMD GPU-specific additions
     * Supports HIP, OpenMP offloading, and other programming models
-    * [Github](https://github.com/ROCm/llvm-project)
-* [AOMP](#AOMP) (AMD OpenMP Compiler) (preview):
+* [AOMP](#AOMP) (AMD OpenMP Compiler) ([Github](https://github.com/ROCm/aomp)) (preview):
     * Specialized for OpenMP target offloading to AMD GPUs
     * Based on the LLVM project with specific optimizations for OpenMP
     * Supports OpenMP 5.0+ features relevant to GPU offloading
     * Currently a development-preview, not yet a full product
-    * [Github](https://github.com/ROCm/aomp)
 * [AOCC](#AOCC) (AMD Optimizing C/C++ Compiler):
     * Primarily focused on AMD CPU optimization
     * Can be used in conjunction with [ROCm](#ROCm) for heterogeneous computing
     * Based on LLVM/Clang with AMD-specific optimizations
     * Closed source
-* hipcc:
+* HIPCC:
     * Compiler wrapper for HIP applications
     * Simplifies compilation process by handling complex flag combinations
-    * Part of the HIP package
+    * Part of the [HIP](#HIP) package and the [ROCm-LLVM](#ROCm-LLVM) project
 
 ### Fortran Compilers
 
 * [AOCC](#AOCC) (AMD Optimizing Fortran Compiler):
     * Based on Flang and LLVM
     * Supports GPU offloading via OpenMP directives
     * Optimized for AMD architectures
-* Flang for ROCm (deprecated):
+* Flang for ROCm ([Github](https://github.com/ROCm/flang)) (deprecated):
     * Part of the LLVM project's Fortran implementation
     * The new Flang implementation (as described in LLVM's blog post) brings improved compatibility and performance
-    * [Github](https://github.com/ROCm/flang)
-
-### Compilers Dependencies
-```mermaid
-graph LR;
-    subgraph Compilers
-        A[ROCm LLVM]
-        B[AOCC]
-        C[HIPCC]
-    end
-    subgraph ROCm Components
-        D[ROCm Core]
-        E[HIP]
-    end
-    A -->|Depends on| D
-    C -->|Depends on| E
-```
 
 ## Developer Tools {: #Developer-Tools }
 
 [ROCm](#ROCm) offers several tools to aid in development, debugging, and performance optimization:
 
-* ROCgdb: Debugger for HIP and OpenCL applications
-    * [Github](https://github.com/ROCm/ROCgdb)
-* ROCProfiler: Performance profiling tool
-    * [Github](https://github.com/ROCm/rocprofiler)
-* rocm-cmake: CMake modules for ROCm
```

Comment:
@Zeldhoron `ROCm-cmake` is still mentioned in mermaid diagram below, but it was removed from the list here